src/dbnode/network/server/tchannelthrift/node/service.go

```diff
@@ -538,6 +538,61 @@ func (s *service) query(ctx context.Context, db storage.Database, req *rpc.Query
 	return result, nil
 }
 
+func (s *service) AggregateTiles(tctx thrift.Context, req *rpc.AggregateTilesRequest) (*rpc.AggregateTilesResult_, error) {
+	db, err := s.startWriteRPCWithDB()
+	if err != nil {
+		return nil, err
+	}
+	defer s.writeRPCCompleted()
+
+	ctx, sp, sampled := tchannelthrift.Context(tctx).StartSampledTraceSpan(tracepoint.AggregateTiles)
+	defer sp.Finish()
+
+	if sampled {
+		sp.LogFields(
+			opentracinglog.String("sourceNameSpace", req.SourceNameSpace),
+			opentracinglog.String("targetNameSpace", req.TargetNameSpace),
```

Comment:
nit: Use "sourceNamespace" and "targetNamespace". We don't camel case namespace.

Reply:
Done.